### PR TITLE
Rename auditd fields for ECS

### PR DESCRIPTION
**Pull request by andrewkroh**
Change `auditd.messages` to `event.original` and `auditd.warnings` to `error.message`.
And also change `user.user_information` from text to keyword.

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -41,6 +41,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Field `file.origin` changed type from `text` to `keyword`. {pull}10544[10544]
 - Rename user fields to ECS in auditd module. {pull}10456[10456]
 - Rename `event.type` to `auditd.message_type` in auditd module because event.type is reserved for future use by ECS. {pull}10536[10536]
+- Rename `auditd.messages` to `event.original` and `auditd.warnings` to `error.message`. {pull}NNNN[NNNN]
```
Comment (webmat) [inline CHANGELOG.next.asciidoc]:
```suggestion
- Rename `auditd.messages` to `event.original` and `auditd.warnings` to `error.message`. {pull}10577[10577]
```

File: auditbeat/module/auditd/_meta/fields.yml [hunk 2]
```diff
@@ -851,15 +851,20 @@
...
       description: >
         An ordered list of the raw messages received from the kernel that
         were used to construct this document. This field is present if an error
         occurred processing the data or if `include_raw_message` is set
         in the config.
     - name: warnings
-      type: keyword
+      type: alias
+      migration: true
+      path: error.message
```
Comment (webmat) [inline auditbeat/module/auditd/_meta/fields.yml]: Not a fan of using `error.message` for debugging output. It's not an actual error message from the source, nor is it a processing error, really, according to the description below.

File: x-pack/auditbeat/module/system/user/_meta/fields.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@
       description: >
           Program to run at login.
     - name: user_information
-      type: text
+      type: keyword
```
Comment (webmat) [inline x-pack/auditbeat/module/system/user/_meta/fields.yml]: 👍

File: auditbeat/module/auditd/_meta/fields.yml (hunk 2 above)
Comment (andrewkroh) [reply auditbeat/module/auditd/_meta/fields.yml]: It's a processing error. For example, message would be included if an expected field were missing while processing the events from the kernel. It holds any of the error messages that are returned by go-libaudit while processing the messages.

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (andrewkroh) [reply CHANGELOG.next.asciidoc]: Thanks 😄

File: auditbeat/module/auditd/_meta/fields.yml (hunk 2 above)
Comment (webmat) [reply auditbeat/module/auditd/_meta/fields.yml]: Gotcha, I'm good with that, then.

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (webmat) [reply CHANGELOG.next.asciidoc]: I still see the wrong thing

**Comment by andrewkroh** [comment]
Will fix. I did a rebase and lost the fix.

**Comment by webmat** [comment]
Only Jenkins failure is metricbeat. Unrelated

**Pull request merged**